docs/abci.md

```diff
@@ -2,11 +2,11 @@
 
 
 
-The [Application BlockChain Interface (ABCI)](https://docs.tendermint.com/master/spec/abci/) defines the boundary between the consensus engine (the blockchain) and an application to be replicated across a number of platforms.
```

Comment:
Tendermint changed urls, they are no longer available in some cases

Reply:
Why is URL checker not failing for this URL?

Reply:
It did locally - it was disabled in CI (enabling now!)